Assets/Global/Scripts/Enemies/FollowingEnemyStates/FollowingState.cs

```diff
@@ -0,0 +1,58 @@
+using UnityEngine;
+
+namespace FollowEnemyStates
+{
+    public class FollowingState : StateBase
+    {
+        public FollowingState(FollowEnemy followEnemy) : base(followEnemy)
+        {
+        }
+        public override void Enter()
+        {
+        }
```

Comment:
Whoops, i forgot to remove the debug.logs, even tho it contained the //TODO: lol.
Thats my bad :P